.changeset/itchy-bulldogs-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-puppetdb': patch
+---
+
+Initial version of PuppetDB plugin.
```

Comment:
Can you include a link to the plugin's README? These changesets turn into release notes so it would make it easier for people ti dive right into this if they are interested